.gitignore

```diff
@@ -151,6 +151,8 @@ GSYMS
 /tools/cub-1.8.0/
 /tools/cub
 /tools/python/
+tools/ngram-1.3.7.tar.gz
```

Comment:
would prefer /tools instead of tools,  for consistency with the other patterns

Reply:
Fixed